website/docs/r/iot_policy.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: aws_iot_policy"
+sidebar_current: "docs-aws-resource-iot-policy"
+description: |-
+  Provides an IoT policy.
+---
```

Comment:
We also need to add a link to the sidebar to make this page discoverable, see https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/aws.erb